### ISSUE TITLE
Add a template tag to get the last invalidation timestamp for given model(s)

### ISSUE THREAD

**Issue by BertrandBordage** [BertrandBordage/django-cachalot#18]
Such a template tag could be extremely useful for partial template caching. Something like:

```HTML+Django
{% load cachalot cache %}

{% get_last_invalidation auth.User yourapp.YourModel as last_invalidation %}
{% cache 86400 cache_key last_invalidation %}
    […]
{% endcache %}
```

Amongst the invalidation timestamps of all the specified model names, the closest from now is returned. If no timestamp is found, we return a naive `now()` instead of `None`.

That way, this part of the template is updated when data changes the table `auth_user` or `yourapp_yourmodel`.

The Python equivalent could be added to the API.

**Comment by BertrandBordage** [comment]
This could also be used in complement of a `last_change = DateTimeField(auto_now=True)`. For example:

```HTML+Django
{% load cachalot %}

{% get_last_invalidation auth.User as last_user_invalidation %}
{% cache 86400 cache_key object.last_change last_user_invalidation %}
  <h3>{{ object }} has been created by {{ object.owner.username }}</h3>
  <div>
    [Imagine here a lot of complex stuff generated
    from <code>object</code> that cost so much time
    that caching it would be divide by 2 the response time]
  </div>
{% endcache %}
```

That way, this part of the template is updated only if `object` has been saved again (and not when any object of the same table is changed, as written in the previous example) or if something changed in your `auth_users` table, but that should happen a lot less often than for the table of `object`.

**Comment by BertrandBordage** [comment]
The first part of this issue (the Python API) has been implemented in 6c136360449900c138da1f9d7fefff134eb8a568.

**Comment by BertrandBordage** [comment]
The last part of this issue has been implemented in f5bfeec59b183bcc4668cf8cb23aa02c53e21e96.
